apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/commands/DeleteCopiedDataCommand.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.commands;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+
+import com.apollocurrency.aplwallet.apl.core.shard.DataTransferManagementReceiver;
+import com.apollocurrency.aplwallet.apl.core.shard.MigrateState;
+import org.slf4j.Logger;
+
+public class DeleteCopiedDataCommand implements DataMigrateOperation {
+    private static final Logger log = getLogger(DeleteCopiedDataCommand.class);
+
+    private DataTransferManagementReceiver dataTransferManagement;
+    private List<String> tableNameList = new LinkedList<>();
```

Comment:
Why we use here LinkedList?

Reply:
Preserve steps/command execution order

Reply:
ArrayList also guarantee order preservation for elements.